src/wazuh_modules/vulnerability_detector/wm_vuln_detector_nvd.c

```diff
@@ -1988,6 +2025,172 @@ int wm_vuldet_report_nvd_vulnerabilities(sqlite3 *db, vu_nvd_report **nvd_report
     return retval ? wm_vuldet_sql_error(db, stmt) : 0;
 }
 
+char *wm_vuldet_clean_version(const char *version) {
+    char *data  = NULL;
+    char *dup   = NULL;
+    char *clean_version = NULL;
+
+    os_strdup(version, dup);
+
+    // Store the data between : and -
+    strtok(dup, ":");
+    data = strtok(NULL, ":");
+    data = strtok((data) ? data : dup, "-");
+
+    os_calloc(1, strlen(data) + 1, clean_version);
+    strcpy(clean_version, data);
```

Comment:
The size of the package's version is handled by SysCollector(`TYPE_LENGTH` and `OS_MAXSTR` macros), so no worries on that side. Furthermore, bufferOverflows are not possible since `clean_version` has the same size as `data` and is NULL-terminated due to `calloc()` zeroing all allocated memory by default. 
If all harmful behaviours are handled correctly, it's not unwise the usage of `strcpy` over `strncpy` for performance reasons; the first one can almost be 30% faster than the second one.